Web/coolprop/LowLevelAPI.rst

```diff
@@ -25,6 +25,46 @@ which yields the output:
 
 .. literalinclude:: snippets/AbstractState1.cxx.output
 
+Here is an example of the shared library usage with Julia wrapper::
+
+    julia> import CoolProp
+
+    julia> PT_INPUTS = CoolProp.get_input_pair_index("PT_INPUTS")
```

Comment:
Are you thinking of something like http://www.coolprop.org/_static/doxygen/html/namespace_cool_prop.html#a58e7d98861406dedb48e07f551a61efb ?